### ISSUE TITLE
Opponent smoke is drawn incorrectly behind level geometry

### ISSUE THREAD

**Issue by rafalp** [dethrace-labs/dethrace#214]
Smoke is drawn behind ice on ice platform level:

<img width="1440" alt="Zrzut ekranu 2022-10-16 o 21 47 13" src="https://user-images.githubusercontent.com/750553/196055237-d755b825-c850-44ba-ad3e-e06279dd23ad.png">

**Comment by rafalp** [comment]
Looks like this is not specific to transparency. Here's four race in Splat Pack, and smike is drawn behind all geometry:

<img width="1440" alt="Zrzut ekranu 2022-10-16 o 22 28 39" src="https://user-images.githubusercontent.com/750553/196056748-9cb75bd3-fd4c-4dbb-9251-8fc7f0a0d8ee.png">

<img width="1440" alt="Zrzut ekranu 2022-10-16 o 22 27 08" src="https://user-images.githubusercontent.com/750553/196056762-04e20c21-2eca-4fcd-a4da-3b639e330453.png">

**Comment by zear** [comment]
This also impacts other effects, e.g. Electro-bastard Ray is only visible on top of car shadows in "Fridge Racer" (3rd race in original Carmageddon) and in the aforementioned "Drown in the Sumps" (4th race from Splat Pak).

**Comment by zear** [comment]
"By the Sea Scythe" is another race where this happens for me. For the sake of completeness, my renderer details:
```
[INFO] GLRenderer_Init OpenGL vendor string: AMD
[INFO] GLRenderer_Init OpenGL renderer string: AMD Radeon Vega 11 Graphics (raven, LLVM 14.0.6, DRM 3.48, 6.0.10-arch2-1)
[INFO] GLRenderer_Init OpenGL version string: 4.6 (Compatibility Profile) Mesa 22.2.3
[INFO] GLRenderer_Init OpenGL shading language version string: 4.60
```

**Comment by dethrace-labs** [comment]
This was fixed in #255
